candidate-standard/openapi/examples/collection.json

```diff
@@ -0,0 +1,625 @@
+{
+    "links": [
+        {
+            "href": "http://www.example.org/edr/collections/",
+            "hreflang": "en",
+            "rel": "self",
+            "type": "application/json"
+        },
+        {
+            "href": "http://www.example.org/edr/collections/",
+            "hreflang": "en",
+            "rel": "alternate",
+            "type": "text/html"
+        },
+        {
+            "href": "http://www.example.org/edr/collections/",
+            "hreflang": "en",
+            "rel": "alternate",
+            "type": "application/xml"
+        }
+    ],
+    "collections": [
+        {
+            "id": "hourly observations",
+            "title": "Hourly Site Specific observations",
+            "description": "Observation data for UK observing sites",
+            "keywords": [
+                "Wind Direction",
+                "Wind Speed",
+                "Wind Gust",
+                "Air Temperature",
+                "Weather",
+                "Relative Humidity",
+                "Dew point",
+                "Pressure",
+                "Pressure Tendancy",
+                "Visibility"
+            ],
+            "links": [
+                {
+                    "href": "http://www.example.org/uk-hourly-site-specific-observations",
+                    "hreflang": "en",
+                    "rel": "service-doc",
+                    "type": "text/html",
+                    "title": ""
+                },
+                {
+                    "href": "http://www.example.org/terms-and-conditions---datapoint#datalicence",
+                    "hreflang": "en",
+                    "rel": "license",
+                    "type": "text/html",
+                    "title": ""
+                },
+                {
+                    "href": "https://www.metoffice.gov.uk/services/data/datapoint/terms-and-conditions---datapoint#termsofservice",
+                    "hreflang": "en",
+                    "rel": "restrictions",
+                    "type": "text/html",
+                    "title": ""
+                },
+                {
```

Comment:
What indicates that this object describes a data query? I believe that "type" attribute is misused here. It is supposed to be a Media/Content Type.